docs/sources/configure/options.md

```diff
@@ -171,12 +171,30 @@ Sets the verbosity level of the process standard output logger.
 Valid log level values are: `DEBUG`, `INFO`, `WARN` and `ERROR`.
 `DEBUG` being the most verbose and `ERROR` the least verbose.
 
+| YAML            | Environment variable  | Type    | Default    |
+| --------------  | --------------------- | ------- | ---------- |
+| `trace_printer` | `BEYLA_TRACE_PRINTER` | string  | `disabled` |
+
+<a id="printer"></a>
+
+Prints any instrumented trace on the standard output (stdout). The value of
+this option specify the format to be used when printing the trace. Valid
+formats are:
+
+| Value         | Description                    |
+|---------------|--------------------------------|
+| `disabled`    | disables the printer           |
+| `text`        | prints a concise line of text  |
+| `json`        | prints a compact JSON object   |
+| `json_indent` | prints an indented JSON object |
+
+
+**The following option has been DEPRECATED.** *Use `trace_printer` instead.*
```

Comment:
I've added this to be explicit about the deprecation - not sure if this is what we usually do, so happy to remove it otherwise.

Reply:
No need to mention the deprecation, we should only have documents for the new option, for the folks already using PRINT_TRACES=1 things will work as usual.